util/SqsMessageHandler.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Reads SQS messages and returns the message if available
+ * @param {Object} params Queue URL and other SQS attributes
+ * @param {Object} sqs AWS SQS service object
+ * @returns {Promise} Returns false if no message was recieved 
+ * or if there was an error, otherwise returns the recieved message
+ */
+function sqsReadHandler(params, sqs) {
```

Comment:
nitpicking here, you decide if u wanna rename it
```suggestion
function readMessageFromSqs(params, sqs) {
```